src/pages/dataTableUtilities/getPageInitialiser.js

```diff
@@ -185,6 +185,7 @@ const stockInitialiser = () => {
     filterDataKeys: ['name', 'code'],
     sortKey: 'name',
     isAscending: true,
+    showAll: true,
```

Comment:
@arjunSussol since seeing zero stock is annoying enough to ask for the feature addition, I think the default could be showAll false. 

However a bigger concern is the user of state variable `showAll`. I think it should be `isShowZeroStocks`. Show is a state where all filters are removed, which is the default state. There is no show all button. What we are doing is adding a filter to default state so the state should represent that filter not show all state. 

Right now there is only one filter so this works for now but if someone add one more filter my argument will start to become more relevant. So we should do this now when the PR is young.

Reply:
@sworup I have tried to implement your suggestion, but it could break the existing workflow.
For toggle, I am using existing `toggleStockOut` reducer which is using by another functionality too.
So, If I rename the state variable `showAll` to something else then the recent toggle as well as existing toggle functionality is not working properly. Either we have to change the code all over or stick with existing.

I preferred existing code if we are using existing function/method.
Later, if we are going to create new reducer with no dependency over existing one then we can create new state variable, but for now it should be fine.